lib/vsc/utils/generaloption.py

```diff
@@ -564,30 +564,30 @@ def set_description_docstring(self):
         except (IndexError, ValueError, AttributeError):
             self.log.debug("set_description_docstring: no docstring found in latest stack globals")
             docstr = None
+            return None
```

Comment:
not a big fan of inline `return` statements, but fine...

Reply:
not a fan of pointless indents